_data/people.yml

```diff
@@ -71,7 +71,7 @@ allen_g:
   topics:
   email: gallen@ncsa.illinois.edu
   homepage: http://www.astro.illinois.edu/people/gdallen
-  
+
```

Comment:
You should configure your editor to keep extra spaces at end of line for the whole of the JLESC site. It will break Markdown!